src/sonic-config-engine/tests/sample_output/py3/arista7260-dualtor.config.bcm

```diff
@@ -0,0 +1,943 @@
+-%}l3_alpm_hit_skip=1
```

Comment:
remove the -%}. How are the tests passing with this?

Reply:
there is a bug, fixed it.